src/py/flwr/superexec/deployment.py

```diff
@@ -144,21 +144,25 @@ def start_run(
             run_id: int = self._create_run(fab_id, fab_version, override_config)
             log(INFO, "Created run %s", str(run_id))
 
-            # Start ServerApp
+            command = [
+                "flower-server-app",
+                "--run-id",
+                str(run_id),
+                "--superlink",
+                str(self.superlink),
+            ]
+
+            if self.flwr_dir:
+                command.append(f"--flwr-dir {self.flwr_dir}")
+
+            if self.root_certificates is None:
+                command.append("--insecure")
+            else:
+                command.append(f"--root-certificates {self.root_certificates}")
```

Comment:
Should we use two appends here? Arg name and arg value are two distinct list entires in the list above